js/core/form/editors/impl/avatar/controllers/DemoAvatarEditorController.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Developer: Oleg Verevkin
+ * Date: 02/19/2016
+ * Copyright: 2009-2016 Comindware®
+ *       All Rights Reserved
+ * Published under the MIT license
+ */
+
+'use strict';
+
+const defaultOptions = {
+    defaultURL: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEgAAABIAQMAAABvIyEEAAAABlBMVEUAAABTU1OoaSf/AAAAAXRSTlMAQObYZgAAAENJREFUeF7tzbEJACEQRNGBLeAasBCza2lLEGx0CxFGG9hBMDDxRy/72O9FMnIFapGylsu1fgoBdkXfUHLrQgdfrlJN1BdYBjQQm3UAAAAASUVORK5CYII='
+};
+
+export default Marionette.Object.extend({
```

Comment:
It would be better to create a base class like in reference editor (BaseAvatarEditorController) and then inherit from it. The base class may not have any logic (pure interface, all methods throw NotImplementedError), but we can decorate it with JSDoc.